src/components/TextField/TextField.tsx

```diff
@@ -29,6 +30,8 @@ export interface TextFieldProps extends BaseTextFieldProps, DebouncedOnChangePro
  * a single line of text. The text displays on the screen in a simple, uniform format.
  */
 class TextField extends React.Component<TextFieldProps & DebouncedOnChangePrivateProps> {
+  private fabricTextFieldElement: ITextField | undefined;
```

Comment:
i believe it's possible for Fabric to return `null` in the ref callback, unless that recently changed? if it could be null, this should be `ITextField | null` and then we should set `this.fabricTextFieldElement = null;` in the constructor

Reply:
> i believe it's possible for Fabric to return null in the ref callback
Still is, in this case `getRef` converts the `null` to `undefined`.

Your suggestion is how I originally had it, but then refactored to this, preferring not to have the constructor if we didn't need it. Do you see an advantage to changing it back?

Reply:
React itself returns a component/node or `null`, so if we replace `null` with `undefined` we're just doing something nonstandard. other YamUI components also use `null` so i'd prefer consistency, but it's not a big deal since this property is private to this component.